media/js/wiki.js

```diff
@@ -847,5 +847,73 @@
     if($('details').length){
         initDetailsTags();
     }
+    
+    function create_jsfiddle(title, htmlCode, cssCode, jsCode) {
+       var $form = $('<form method="post" target="_blank" action="https://jsfiddle.net/api/post/library/pure/">'
+            + '<input type="hidden" id="html" name="html" />'
```

Comment:
Do we need any of these `id` attributes?  These seem unnecessary.  Instead of `$form.find('#html')`, can you do `$form.find('input[name=html')`?